src/main/java/org/medicmobile/webapp/mobile/MrdtSupport.java

```diff
@@ -37,7 +37,8 @@ String process(int requestCode, int resultCode, Intent i) {
 				try {
 					byte[] data = i.getByteArrayExtra("data");
 					String base64data = Base64.encodeToString(data, Base64.NO_WRAP);
-					return safeFormat("angular.element(document.body).injector().get('AndroidApi').v1.mrdtResponse('\"%s\"')", base64data);
+					long timeTaken = i.getLongExtra("timeTaken", 0);
+					return safeFormat("angular.element(document.body).injector().get('AndroidApi').v1.mrdtResponse('\"%s|%s\"')", String.valueOf(timeTaken), base64data);
```

Comment:
Can we use a `get('AndroidApi').v2` to make this backward compatible?

Reply:
A v2 would not help I think cause the bulk of the logic is in the enketo widget .. I have implemented what I believe is a simple change to ensure that 3.6 works with the current version of the wrapper. I needed to swap the positions of timeTaken and data (image) to ensure that it works

Reply:
A project running a version of the webapp < 3.6 would break mRDT if they upgrade to the new container. I think the change in the container needs to be made backward compatible, or it will really cause issues.

Have you thought about passing this data separately instead of changing the data schema that is passed to `mrdtResponse`? You could make the same call to `mrdtResponse` (not breaking its backward compatibility) and also make a new call to `mrdtTimestampResponse` or alike.